### DISCUSSION TITLE
Feature Request: Fuel Economy Displays for Summary Card and Details

### DISCUSSION THREAD

**Discussion by KLMors3** [AlfHou/hammond#14] [Ideas]
Howdy, I posted a feature request under akhilrex/hammond but upon looking through other discussions, it appears that the development has either stalled or stopped 😐 

Is this request something that y'all can accommodate? I'll happily swap repos for an actively developed one. 

I'm not familiar with GO but I do dabble with html/css and designing layouts for tools I use regularly, so I'm sure I'll have other functionality and design ideas. 

Here's the original request with photos and detail. 
https://github.com/akhilrex/hammond/issues/119

**Comment by AlfHou** [comment]
Hi! Thank you for the great suggestion. I've opened a issue/feature request for it in this repo: https://github.com/AlfHou/hammond/issues/28. I'll try to get to it once I have the time. First there are some cleanup tasks I need to get done (a lot of dependencies out of date etc.)
